client/src/components/AddProduct/AddProduct.tsx

```diff
@@ -0,0 +1,151 @@
+import { CssBaseline, Dialog, DialogContent, Grid, Icon, Typography } from '@material-ui/core';
+import React, { useState } from 'react';
+import useStyles from './useStyles';
+import { FormikHelpers } from 'formik';
+import AddProductForm from './AddProductForm/AddProductForm';
+import AddProductPreview from './AddProductPreview/AddProductPreview';
+import { ProductDetails } from '../../interface/ProductDetails';
+import { useSnackBar } from '../../context/useSnackbarContext';
+import { getAllList } from '../../helpers/APICalls/lists';
+
+interface Props {
+  showAddProductModal: boolean;
+  setShowAddProductModal: React.Dispatch<React.SetStateAction<boolean>>;
+}
+
+const getProductLists = async () => {
+  // FIXME uncomment to fetch from ami
```

Comment:
is this waiting for certain piece of code? i would want to merge commented out block like this into main branch

Reply:
i'm waiting for `lists features`